src/software/proto/message_translation/ssl_detection_message_translator.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include <memory>
+
+#include "software/proto/messages_robocup_ssl_detection.pb.h"
+#include "software/time/timestamp.h"
+#include "software/world/ball_state.h"
+#include "software/world/robot_state.h"
+
+/**
+ * Creates a DetectionBall from the given BallState.
+ *
+ * @param ball The ball to convert to a DetectionBall
+ *
+ * @return A DetectionBall representing the given BallState
+ */
+std::unique_ptr<SSL_DetectionBall> createDetectionBall(const BallState& ball);
```

Comment:
can we toss SSL in the function signature so we know its obvious its not our internal msg type

ditto for the rest

Reply:
done